Multiline templates in YAML config file are parsed wrong

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

After upgrading to [v14.2.6](https://github.com/JanDeDobbeleer/oh-my-posh/releases/tag/v14.2.6) I guess, some [templates](https://gist.github.com/qyurila/b26614f65d9a79577b0fc08668dcdced#file-emodipt-extend-mod-omp-yaml-L55-L62) of my configuration file started to be parsed wrong. Then I found that it's a problem with goccy/go-yaml and created an issue there: goccy/go-yaml#344 (I noticed that the actual package used is a fork tho)

In my opinion, it would be good to revert YAML parser until the issue is resolved if there are no other serious problems. Rich multiline string support is (despite terrible) one of (if not only) the good reasons of using YAML for config because it makes dealing with templates somewhat tolerable, and it seems that the new parser impaired it...

### Theme

[My own modification of emodipt-extend](https://gist.github.com/qyurila/b26614f65d9a79577b0fc08668dcdced)

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 14.2.7

Segments:

ConsoleTitle(false) -   0 ms -
ytm(false)          -  25 ms -
shell(true)         -   0 ms -  oh-my-posh
git(true)           -  53 ms - \  master ↑1 \ \ \ \ \
node(true)          -  24 ms -  18.14.0
python(false)       -   0 ms -
rust(false)         -   0 ms -
go(false)           -   0 ms -
dotnet(false)       -   1 ms -
time(true)          -   0 ms -  17:00:04
path(true)          -   1 ms -  ~\Dev\Personal\Plugins\obsidian-rentry-paster
exit(false)         -   0 ms -
executiontime(true) -   0 ms -  0ms
root(false)         -   0 ms -
text(true)          -   0 ms - ❯

Run duration: 118.3114ms

Cache path: C:\Users\Hyeon\AppData\Local\oh-my-posh

Config path: C:\Users\Hyeon\Documents\_Backups\emodipt-extend-mod.omp.yaml

Logs:

[DEBUG] 17:00:04.016 shell.go:Getenv:347 → C:\Users\Hyeon\AppData\Local
[TRACE] 17:00:04.016 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 17:00:04.017 shell_windows.go:CachePath() - 508µs
[DEBUG] 17:00:04.017 shell.go:Getenv:347 → C:\Users\Hyeon\Documents\_Backups\emodipt-extend-mod.omp.yaml
[TRACE] 17:00:04.017 shell.go:Getenv(POSH_THEME) - 0s
[DEBUG] 17:00:04.024 shell.go:Shell:592 → process name: oh-my-posh.exe
[TRACE] 17:00:04.024 shell.go:Shell() - 7.3243ms
[TRACE] 17:00:04.024 shell.go:resolveConfigPath() - 7.3243ms
[TRACE] 17:00:04.024 shell.go:Init() - 7.8323ms
[TRACE] 17:00:04.024 shell.go:Flags() - 0s
[TRACE] 17:00:04.027 config.go:loadConfig() - 2.5929ms
[TRACE] 17:00:04.027 shell.go:Flags() - 0s
[DEBUG] 17:00:04.027 shell.go:Getenv:347 → NO DATA
[TRACE] 17:00:04.027 shell.go:Getenv(OMP_CACHE_DISABLED) - 0s
[TRACE] 17:00:04.027 shell_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.027 shell_windows.go:WindowsRegistryKeyValue:206 → ColorizationColor(DWORD): 0xC40099BC
[TRACE] 17:00:04.027 shell.go:Flags() - 0s
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.027 shell.go:Pwd:375 → NO DATA
[TRACE] 17:00:04.027 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.027 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.027 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.027 shell.go:HTTPRequest:636 ↓
    GET /query HTTP/1.1
    Host: 127.0.0.1:9863
    User-Agent: Go-http-client/1.1
    Accept-Encoding: gzip


[ERROR] 17:00:04.053 shell.go:HTTPRequest:640 → Get "http://127.0.0.1:9863/query": context deadline exceeded
[TRACE] 17:00:04.053 shell.go:HTTPRequest(http://127.0.0.1:9863/query) - 25.767ms
[TRACE] 17:00:04.053 shell.go:Flags() - 0s
[DEBUG] 17:00:04.053 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.053 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.053 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:Shell() - 0s
[TRACE] 17:00:04.053 shell.go:Flags() - 0s
[TRACE] 17:00:04.053 shell_windows.go:Root() - 0s
[TRACE] 17:00:04.053 shell.go:Shell() - 0s
[TRACE] 17:00:04.053 shell.go:ErrorCode() - 0s
[TRACE] 17:00:04.053 shell_windows.go:IsWsl() - 0s
[DEBUG] 17:00:04.053 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.053 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.053 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.053 shell.go:User:496 → Hyeon
[TRACE] 17:00:04.053 shell.go:User() - 0s
[DEBUG] 17:00:04.053 shell.go:Host:508 → Hyeon-PC-23
[TRACE] 17:00:04.053 shell.go:Host() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.053 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.053 shell.go:Shell() - 0s
[DEBUG] 17:00:04.053 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.053 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.053 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[TRACE] 17:00:04.053 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.065 shell.go:CommandPath:547 → C:\Users\Hyeon\scoop\shims\git.exe
[TRACE] 17:00:04.065 shell.go:CommandPath(git.exe) - 11.4676ms
[TRACE] 17:00:04.065 shell.go:HasCommand(git.exe) - 11.4676ms
[DEBUG] 17:00:04.065 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.065 shell.go:Pwd() - 0s
[TRACE] 17:00:04.065 shell.go:HasParentFilePath(.git) - 0s
[DEBUG] 17:00:04.065 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.065 shell.go:Getenv:347 → NO DATA
[TRACE] 17:00:04.065 shell.go:Getenv(POSH_GIT_STATUS) - 0s
[DEBUG] 17:00:04.065 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.065 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.105 shell.go:RunCommand:526 ↓
    # branch.oid f8330c621a68216529edd1ddaa8c0bc932df5af7
    # branch.head master
    # branch.upstream origin/master
    # branch.ab +1 -0
[TRACE] 17:00:04.105 shell.go:RunCommand(git.exe -C C:/Users/Hyeon/Dev/Personal/Plugins/obsidian-rentry-paster --no-optional-locks -c core.quotepath=false -c color.status=false status -unormal --branch --porcelain=2) - 40.5744ms
[DEBUG] 17:00:04.105 shell.go:HasFolder:435 → false
[TRACE] 17:00:04.105 shell.go:HasFolder(C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster\.git/rebase-merge) - 0s
[DEBUG] 17:00:04.105 shell.go:HasFolder:435 → false
[TRACE] 17:00:04.105 shell.go:HasFolder(C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster\.git/rebase-apply) - 0s
[DEBUG] 17:00:04.105 shell.go:HasFilesInDir:406 → false
[TRACE] 17:00:04.105 shell.go:HasFilesInDir(MERGE_MSG) - 0s
[DEBUG] 17:00:04.105 shell.go:HasFilesInDir:406 → false
[TRACE] 17:00:04.105 shell.go:HasFilesInDir(CHERRY_PICK_HEAD) - 0s
[DEBUG] 17:00:04.105 shell.go:HasFilesInDir:406 → false
[TRACE] 17:00:04.105 shell.go:HasFilesInDir(REVERT_HEAD) - 0s
[DEBUG] 17:00:04.105 shell.go:HasFilesInDir:406 → false
[TRACE] 17:00:04.105 shell.go:HasFilesInDir(sequencer/todo) - 0s
[TRACE] 17:00:04.105 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.106 shell.go:TemplateCache() - 0s
[ERROR] 17:00:04.106 shell.go:FileContent:460 → open C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster\.git/logs/refs/stash: The system cannot find the file specified.
[TRACE] 17:00:04.106 shell.go:FileContent(C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster\.git/logs/refs/stash) - 528.5µs
[TRACE] 17:00:04.106 shell.go:Shell() - 0s
[TRACE] 17:00:04.106 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.106 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.107 shell.go:TemplateCache() - 0s
[DEBUG] 17:00:04.107 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.107 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.107 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.107 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.107 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.107 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.107 shell_windows.go:56 → C:\Users\Hyeon
[DEBUG] 17:00:04.107 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.107 shell.go:Pwd() - 0s
[TRACE] 17:00:04.107 shell.go:HasFiles(*.js) - 0s
[DEBUG] 17:00:04.112 shell.go:CommandPath:547 → C:\Users\Hyeon\scoop\apps\nodejs-lts\current\node.exe
[TRACE] 17:00:04.112 shell.go:CommandPath(node) - 5.1337ms
[TRACE] 17:00:04.112 shell.go:HasCommand(node) - 5.1337ms
[DEBUG] 17:00:04.131 shell.go:RunCommand:526 → v18.14.0
[TRACE] 17:00:04.131 shell.go:RunCommand(node --version) - 18.6866ms
[TRACE] 17:00:04.131 shell.go:TemplateCache() - 0s
[DEBUG] 17:00:04.131 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.131 shell.go:Pwd() - 0s
[ERROR] 17:00:04.131 shell.go:FileContent:460 → open C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster\.nvmrc: The system cannot find the file specified.
[TRACE] 17:00:04.131 shell.go:FileContent(.nvmrc) - 0s
[TRACE] 17:00:04.131 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.132 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.132 shell.go:Shell() - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.132 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[DEBUG] 17:00:04.132 shell.go:Getenv:347 → NO DATA
[TRACE] 17:00:04.132 shell.go:Getenv(VIRTUAL_ENV) - 0s
[DEBUG] 17:00:04.132 shell.go:Getenv:347 → NO DATA
[TRACE] 17:00:04.132 shell.go:Getenv(CONDA_ENV_PATH) - 0s
[DEBUG] 17:00:04.132 shell.go:Getenv:347 → NO DATA
[TRACE] 17:00:04.132 shell.go:Getenv(CONDA_DEFAULT_ENV) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.132 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.rs) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(Cargo.toml) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(Cargo.lock) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.132 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.go) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(go.mod) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.132 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.132 shell_windows.go:56 → C:\Users\Hyeon
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.cs) - 518.8µs
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.csx) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.vb) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.sln) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.132 shell.go:HasFiles(*.slnf) - 0s
[DEBUG] 17:00:04.132 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.132 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(*.csproj) - 519.4µs
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(*.vbproj) - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(*.fs) - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(*.fsx) - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(*.fsproj) - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:HasFiles(global.json) - 0s
[TRACE] 17:00:04.133 shell.go:Flags() - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.133 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.133 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.133 shell.go:Shell() - 0s
[TRACE] 17:00:04.133 shell.go:Flags() - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.133 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell.go:Shell() - 0s
[TRACE] 17:00:04.133 shell.go:Shell() - 0s
[DEBUG] 17:00:04.133 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:GOOS() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[TRACE] 17:00:04.133 shell.go:PathSeparator() - 0s
[DEBUG] 17:00:04.133 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.133 shell.go:Pwd() - 0s
[TRACE] 17:00:04.133 shell_windows.go:IsWsl() - 0s
[TRACE] 17:00:04.133 shell.go:StackCount() - 0s
[DEBUG] 17:00:04.134 win32_windows.go:isWriteable:299 → not current user or in group
[DEBUG] 17:00:04.134 win32_windows.go:isWriteable:303 → current user is member of S-1-5-32-544
[DEBUG] 17:00:04.134 win32_windows.go:isWriteable:311 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 17:00:04.134 win32_windows.go:isWriteable:313 → user has write access
[TRACE] 17:00:04.134 shell_windows.go:DirIsWritable() - 552.7µs
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:Shell() - 0s
[TRACE] 17:00:04.134 shell.go:Flags() - 0s
[DEBUG] 17:00:04.134 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.134 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.134 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.134 shell.go:GOOS() - 0s
[TRACE] 17:00:04.134 shell.go:ErrorCode() - 0s
[TRACE] 17:00:04.134 shell.go:ErrorCode() - 0s
[DEBUG] 17:00:04.134 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.134 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.134 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.134 shell.go:GOOS() - 0s
[TRACE] 17:00:04.134 shell.go:ExecutionTime() - 0s
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:Shell() - 0s
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:Flags() - 0s
[DEBUG] 17:00:04.134 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.134 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.134 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.134 shell.go:GOOS() - 0s
[TRACE] 17:00:04.134 shell_windows.go:Root() - 0s
[DEBUG] 17:00:04.134 shell.go:Pwd:355 → C:\Users\Hyeon\Dev\Personal\Plugins\obsidian-rentry-paster
[TRACE] 17:00:04.134 shell.go:Pwd() - 0s
[DEBUG] 17:00:04.134 shell_windows.go:56 → C:\Users\Hyeon
[TRACE] 17:00:04.134 shell.go:GOOS() - 0s
[TRACE] 17:00:04.134 shell.go:TemplateCache() - 0s
[TRACE] 17:00:04.134 shell.go:Shell() - 0s
[DEBUG] 17:00:04.135 shell.go:Getenv:347 → C:\Users\Hyeon\AppData\Local
[TRACE] 17:00:04.135 shell.go:Getenv(LOCALAPPDATA) - 0s
[TRACE] 17:00:04.135 shell_windows.go:CachePath() - 0s
[TRACE] 17:00:04.135 shell.go:Flags() - 0s
```

--- a/src/go.mod
+++ b/src/go.mod
@@ -100,4 +100,4 @@ replace github.com/shirou/gopsutil/v3 v3.22.10 => github.com/jandedobbeleer/gops
 
 replace github.com/gookit/config/v2 v2.1.8 => github.com/jandedobbeleer/config/v2 v2.1.8-1
 
-replace github.com/goccy/go-yaml v1.9.8 => github.com/jandedobbeleer/go-yaml v1.9.8-2
+replace github.com/goccy/go-yaml v1.9.8 => github.com/jandedobbeleer/go-yaml v1.9.8-3

--- a/src/platform/shell_unix.go
+++ b/src/platform/shell_unix.go
@@ -59,6 +59,7 @@ func (env *Shell) TerminalWidth() (int, error) {
 func (env *Shell) Platform() string {
 	const key = "environment_platform"
 	if val, found := env.Cache().Get(key); found {
+		env.Debug(val)
 		return val
 	}
 	var platform string
@@ -67,6 +68,7 @@ func (env *Shell) Platform() string {
 	}()
 	if wsl := env.Getenv("WSL_DISTRO_NAME"); len(wsl) != 0 {
 		platform = strings.Split(strings.ToLower(wsl), "-")[0]
+		env.Debug(platform)
 		return platform
 	}
 	platform, _, _, _ = host.PlatformInformation()

